install_config/topics/configuring_for_aws_registry-proc.adoc

```diff
@@ -80,12 +80,18 @@ openshift_hosted_registry_pullthrough=true
 openshift_hosted_registry_acceptschema2=true
 openshift_hosted_registry_enforcequota=true
 openshift_hosted_registry_replicas=3
+# AWS Registry Encryption Configuration
+openshift_hosted_registry_storage_s3_encrypt
+openshift_hosted_registry_storage_s3_kmskeyid
+secure: default ('true') <5>
+v4auth: default ('true') <6>
```

Comment:
We don't have `secure` not `v4auth` in Ansible.

Reply:
Removed `secure` and `v4auth` here.